api/admin/controller.py

```diff
@@ -1452,6 +1452,7 @@ def collections(self):
             else:
                 return NO_PROTOCOL_FOR_NEW_SERVICE
 
+        collection.name = name
```

Comment:
This is an unrelated bug I found - if you changed a Collection name in the admin interface it didn't actually change.